src/python/pants/backend/python/target_types.py

```diff
@@ -630,7 +643,7 @@ class SetupPyCommandsField(StringSequenceField):
 
 
 class PythonDistribution(Target):
-    """A publishable Python distribution."""
+    """A publishable Python distribution (e.g. wheel)."""
```

Comment:
I somehow want to express `setup.py`.

Reply:
A publishable Python setuptools distribution (e.g., an sdist or wheel).